File: JavaScript/tablasDeMultiplicar.js
let ingresaNumeroHastaQueTabla = parseInt(prompt("Ingresa hasta que tabla deseas multiplicar: "))
let ingresaHastaDondeSeMultiplicara = parseInt(prompt("IngresaIngresa hasta que número vas a multiplicar 1 _ ?: "))

for (let i = 1; i <= ingresaNumeroHastaQueTabla; i++) {
    
    for (let x = 1; x <= ingresaHastaDondeSeMultiplicara; x++) {
        document.write(`${i} x ${x} = ${i*x} <br>`)
    }
    document.write("--------------------- <br>")
}

File: C++/tablasDeMultiplicar.cpp
#include <iostream>
using namespace std;

int main()
{
    int ingresaNumeroHastaQueTabla;

    cout << "Ingresa hasta que tabla deseas multiplicar: ";
    cin >> ingresaNumeroHastaQueTabla;

    for (int x = 1; x <= ingresaNumeroHastaQueTabla; x++)
    {
        for (int i = 1; i <= 12; i++)
        {
            cout << x << " x " << i << " = " << x * i << endl;
        }
        cout << "------------------------" << endl;
    }
}